code/modules/surgery/organs/lungs.dm

```diff
@@ -345,6 +345,39 @@
 	icon_state = "lungs-c"
 	origin_tech = "biotech=4"
 	status = ORGAN_ROBOT
+	var/species_state = "human"
+	var/vox_oxygen_bonus = 0.05 //For upgraded robotic lungs
+
+/obj/item/organ/internal/lungs/cybernetic/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>[src] is configured for [species_state] standards of atmosphere.</span>"
+
+/obj/item/organ/internal/lungs/cybernetic/multitool_act(mob/user, obj/item/I)
+	. = TRUE
+	if(!I.use_tool(src, user, 0, volume = I.tool_volume))
+		return
+	switch(species_state)
+		if("human")
+			safe_oxygen_min = 0
+			safe_oxygen_max = vox_oxygen_bonus
+			safe_nitro_min = 16
+			oxy_damage_type = TOX
+			user.show_message("You configure [src] to replace vox lungs.")
```

Comment:
please add a span class for these. and, why are you using show_message instead of to_chat?